engine/crates/engine-v2/src/sources/graphql/request.rs

```diff
@@ -23,6 +23,15 @@ pub trait ResponseIngester: Send {
     ) -> impl Future<Output = Result<(GraphqlResponseStatus, SubgraphResponse), ExecutionError>> + Send;
 }
 
+impl<T> ResponseIngester for T
+where
+    T: FnOnce(Bytes) -> Result<(GraphqlResponseStatus, SubgraphResponse), ExecutionError> + Send,
+{
+    async fn ingest(self, bytes: Bytes) -> Result<(GraphqlResponseStatus, SubgraphResponse), ExecutionError> {
+        self(bytes)
+    }
+}
```

Comment:
Is this even used anymore?  Think I forgot to remove it.

Reply:
not used, but not a big deal to keep it though imho 🤷